php_memcached.c

```diff
@@ -332,7 +332,11 @@ PHP_INI_BEGIN()
 	MEMC_SESSION_INI_ENTRY("lock_wait_max",          "2000",       OnUpdateLongGEZero,     lock_wait_max)
 	MEMC_SESSION_INI_ENTRY("lock_retries",           "5",          OnUpdateLong,           lock_retries)
 	MEMC_SESSION_INI_ENTRY("lock_expire",            "0",          OnUpdateLongGEZero,     lock_expiration)
+#if defined(LIBMEMCACHED_VERSION_HEX) && LIBMEMCACHED_VERSION_HEX < 0x01000018
+	MEMC_SESSION_INI_ENTRY("binary_protocol",        "0",          OnUpdateBool,           binary_protocol_enabled)
```

Comment:
If this isn't already mentioned in the (package.xml) changelog, could you mention it?

- E.g. someone could upgrade memcached, not notice that the default session binary/text protocol varied based on libmemcached version, use Memcached with something that only supports text protocol(twemproxy), and then upgrade libmemcached, then see something break.
- Or they could have libmemcached and a proxy only working with binary protocol, but I'm not aware of proxies like that.